versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.12.0
+DATE TBD
+
+### Improvements
+- Use `ingressClassName` field for ingresses instead of the deprecated `kubernetes.io/ingress.class` annotation
```

Comment:
is this a breakign change?  if so, we should add a section explicitly calling those out.

Reply:
I think it's a ~slight breaking change but worth documenting yeah:

> Ingresses using the `kubernetes.io/ingress.class` annotation will be automatically updated by the okteto webhook and saved as `spec.ingressClassName` with the annotation removed